WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -477,6 +481,7 @@ public boolean onPrepareOptionsMenu(Menu menu) {
                         } else {
                             saveMenuItem.setTitle(R.string.update_verb);
                         }
+                        mSaveWithoutLeaving = true;
```

Comment:
I'd expect to remain in the editor when tapping the Update button to save changes to a published post.